xl-devops-platform/xebialabs/config.yaml.tmpl

```diff
@@ -0,0 +1,26 @@
+blueprint:
+  current-repository: XL Blueprints
+  repositories:
+  - name: XL Blueprints
+    type: http
+    url: https://dist.xebialabs.com/public/blueprints/
+blueprint-repository:
```

Comment:
This is old config and should not be used, just `blueprint` context above is enough